server/src/com/openrsc/server/net/rsc/handlers/ItemUseOnObject.java

```diff
@@ -133,10 +133,17 @@ public void handlePacket(Packet packet, Player player) throws Exception {
 				item = player.getCarriedItems().getEquipment().get(slotID - Inventory.MAX_SIZE);
 			} else
 				item = player.getCarriedItems().getInventory().get(slotID);
-			if (object.getType() == 1 || item == null || item.getItemStatus().getNoted()) { // This
+			if (object.getType() == 1 || item == null) { // This
 				player.setSuspiciousPlayer(true, "null item or object");
 				return;
 			}
+
+			// Players can't use notes on anything
```

Comment:
Players can use notes on bankers. Even though with context of the codeblock you're in it makes sense, should be more specific & say "players currently can't use notes on any scenery".